code_design/guard_clauses.md

```diff
@@ -0,0 +1,27 @@
+# Replace if/else conditionals with guard clauses
+
+When doing a lot of nested conditionals you may end facing a code like this:
+
+```ruby
+def result_with_nested_conditionals
+  if condition_one
+    return result_one
+  elsif condition_two
+    return result_two
+  else
+    return result_three
+  end
```

Comment:
I think it's not one just because of the elsif keyword. 
If we rewrite the code, removing the elsif and putting an else with a new if/else inside, it will be a nested conditional. In my view, the elsif is just a more readable nested conditional.